armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -835,10 +851,22 @@ def dischargeSwap(self, incoming, outgoing):
             :id: I_ARMI_SHUFFLE_STATIONARY1
             :implements: R_ARMI_SHUFFLE_STATIONARY
 
```

Comment:
```suggestion
            :implements: R_ARMI_SHUFFLE_STATIONARY

            This is the implementation of the stationary requirement for the 
            discharge swap.
```

Reply:
As I said above. I think the correct place for this sort of idea would be the title of the `impl` tag.

Reply:
https://github.com/terrapower/armi/pull/1589/commits/39d55597b00ba43df7be17ad69db9aedcdc0889d